.github/workflows/update.yml

```diff
@@ -39,5 +39,11 @@ jobs:
           K_VERSION=$(poetry run python3 -c 'import pyk; print(pyk.__version__)')
           echo ${K_VERSION} > deps/k_release
           git add deps/k_release && git commit -m "deps/k_release: sync release file version ${K_VERSION}" || true
+      - name: 'Update Nix flake inputs'
+        run: |
+          PYKWASM_VERSION=v"$(cat deps/kwasm_release)"
+          sed -i 's!    wasm-semantics.url = "github:runtimeverification/wasm-semantics/v[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]\+"!    wasm-semantics.url = "github:runtimeverification/wasm-semantics/'"${PYKWASM_VERSION}"'"!' flake.nix
```

Comment:
Do the `+` chars need to be escaped here?

Reply:
Good question... i simpliy copied the code from one of the other repositories. See:

https://github.com/search?q=org%3Aruntimeverification%20%5B%5B%3Adigit%3A%5D%5D%5C%2B%5C.&type=code